src/app/Http/Controllers/ArticleController.php

```diff
@@ -51,7 +53,7 @@ public function store(ArticleRequest $request): Factory|View|Application|Redirec
 
     public function show(Article $article): Factory|View|Application|RedirectResponse
     {
-        return view('pages.articles.show', ['article' => $article]);
+        return view('pages.articles.show', ['article' => $article, 'revisions' => Revision::all(), 'users' => User::all()]);
```

Comment:
'Revisions' is the revisions of the current article. Not all of them. You don't need a separate revisions variable becuse you can get them with `$article->revisions`